src/Symfony/Component/Console/Command/Command.php

```diff
@@ -65,6 +70,17 @@ public static function getDefaultName()
         return $class === $r->class ? static::$defaultName : null;
     }
 
+    /**
+     * @return string|null The default command description or null when no default description is set
+     */
+    public static function getDefaultDescription(): ?string
+    {
+        $class = static::class;
+        $r = new \ReflectionProperty($class, 'defaultDescription');
```

Comment:
Didn't you removed this? 
If I understand well, it prevent a grand-child class to inherit the description from the child class. This could be an issue for the `defaultName` which should be unique, but I think this overhead is not needed in description.

Reply:
Yes, and I added it back. This makes both properties behave the same, and not being inherited, which is the correct behavior to me.